doc/overview.md

```diff
@@ -0,0 +1,149 @@
+- Activate your 3scale account
+- Log in to your 3scale Admin portal
+
+Follow the next steps to configure your API gateway in no time:
+
+### 1. Declare your private base URL
+
+Go to the **API** tab and select the default API, then click on **Integration**.
+
+Add the **private base URL**, which is the endpoint host of your API backend. For instance, if you were Twitter the Private Base URL would be ```https://api.twitter.com/```, or if you are the owner of the Sentiment API it would be ```http://api-sentiment.3scale.net/```.
+
+The gateway will redirect all traffic to your API backend after all authentication, authorization, rate limits and statistics have been processed.
+
+### Step 2: Update the hosted gateway
+
+To update the **hosted gateway** you just have to **save** the settings (API backend, Advanced settings etc.) by clicking on the **Update & Test Staging Configuration** button in the lower right part of the page. This process will deploy your gateway configuration (at this stage the default configuration) to 3scale's hosted gateway.
+
+### Step 3: Get a set of sample credentials
+
+Go to the **Applications** tab, click on any application using the service you're configuring, and get the **API credentials**. If you do not have applications/ users yet, you can create an application yourself (from the details page of any individual developer account), the credentials will be generated automatically.
+
+Typically the credentials will be a ```user_key``` or the pair ```app_id/app_key``` depending on the authentication mode you are using (note that the 3scale hosted gateway does not currently support OAuth, however you can configure it if you are using the self-managed gateway integration.
+
+### Step 4: Get a working request
+
+We are almost ready to roll. Go to your browser (or command-line curl) and do a request to your own API to check that everything is working on your end.
+
+For instance it could be something like this:
+
+```
+http://api-sentiment.3scale.net/v1/word/good.json
+```
+Note that you are not using 3scale's gateway yet. You are just getting a working example that will be used in the next step.
+
+### Step 5: Get a working request to your API
+
+Now do the same request but replacing your **private base URL** (in the example ```http://api-sentiment.3scale.net:80```) by your hosted endpoint (e.g. if you were Twitter, you would need to change the ```https://api.twitter.com``` to ```https://api-xxxxxxxxxxxxx.staging.apicast.io```). You also have to add the parameters to pass the credentials that you just copied.
+
+Continuing the example in this tutorial it would be something like:
+
+```
+https://api-2445581436380.staging.apicast.io:443/v1/word/good.json?user_key=YOUR_USER_KEY
+```
+
+If you execute this request you will get the same result as in Step 4. However, this time the request will go through the 3scale hosted gateway.

```

Comment:
"...and be authorized based on your user_key"

To check that you can repeat the request, removing the user_key query parameter and it should fail.